emojivoto/README.md

```diff
@@ -75,3 +75,52 @@ If you'd like to run the bot manually:
 export WEB_HOST=localhost:8080 # replace with your web location
 go run emojivoto-web/cmd/vote-bot/main.go
 ```
+
+## Local Development
+
+### Emojivoto webapp
+
+This app is written with React and bundled with webpack.
+Use the following to run the emojivoto go services and develop on the frontend.
+
+Start the voting service
+```
+export GRPC_PORT=8081
+go run emojivoto-voting-svc/cmd/server.go
+```
+
+[In a separate teminal window] Start the emoji service
+```
+export GRPC_PORT=8082
+go run emojivoto-emoji-svc/cmd/server.go
```

Comment:
what do you think about:
```
GRPC_PORT=8082 go run emojivoto-emoji-svc/cmd/server.go
```
... it's easier to copy and paste, and won't pollute terminals with vars?

Reply:
I like it!